dev-tools/scripts/buildAndPushRelease.py

```diff
@@ -108,37 +107,26 @@ def prepare(root, version, gpgKeyID, gpgPassword):
   print('  Check DOAP files')
   checkDOAPfiles(version)
 
-  print('  ant -Dtests.badapples=false clean validate documentation-lint test')
-  run('ant -Dtests.badapples=false clean validate documentation-lint test')
+  print('  ./gradlew -Dtests.badapples=false clean check')
+  run('./gradlew -Dtests.badapples=false clean check')
 
   open('rev.txt', mode='wb').write(rev.encode('UTF-8'))
   
-  print('  lucene prepare-release')
-  os.chdir('lucene')
-  cmd = 'ant -Dversion=%s' % version
+  print('  prepare-release')
+  cmd = './gradlew -Dversion=%s clean' % version
```

Comment:
This must be `-Dversion.release=%s`, as gradle has some reserved meaning of `version`.